### PR TITLE
@ashfurrow => Adds support for showing the SOLD info for works that are inquirable

### LINKED ISSUE #314: Wrong Artwork status
I just noticed that on the iOS app we are not specifying the artwork status (e.g. 'Not for Sale', 'Sold' etc). I receive inquiries from the app users on works that are marked 'sold' publicly, but because this info is not visible to them on an iPhone, they still inquire. 
An example is this work. 

Can we add availability status to each work on the app?

### PR DESCRIPTION
**Pull request by orta**
but sold - fixes #314

BTW @alloy this test is failing for me consistently: https://github.com/artsy/eigen/blob/bb6e152ee8c067aa452ae6c81b9d204870d3a356/Artsy%20Tests/ARArtworkRelatedArtworksViewTests.m#L203

### CODE REVIEW

File: Artsy/Classes/Views/ARArtworkActionsView.m [hunk 1]
```diff
@@ -210,15 +211,15 @@ - (void)setEnabled:(BOOL)enabled {
 - (BOOL)showNotForSaleLabel
 {
     return self.artwork.inquireable.boolValue
-    && !self.artwork.sold.boolValue
```
Comment (orta) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: just removes the `!` - this function wasn't being used anywhere in the app

**Comment by ashfurrow** [comment]
Snazzy.

**Comment by ashfurrow** [comment]
CI failure is in related artworks test – doesn't look related.

**Comment by alloy** [comment]
@orta Yeah that seems to be [the same](https://github.com/artsy/eigen/pull/294#issuecomment-85069144) that @1aurabrown reported, I’ll have a look.

**Pull request merged**